port/common/miniz/miniz_export.h

```diff
@@ -0,0 +1,50 @@
+// Golioth comment:
+//
+// This file is required by the miniz library, but is not part of the
+// source-controlled files in that library.
+//
+// Normally it is generated at build time in miniz's CMakeLists.txt.
+// Since we're not using that CMakeLists.txt, we will just add this dir
+// to the include path of our build.
+
+#ifndef MINIZ_EXPORT_H
+#define MINIZ_EXPORT_H
+
+#ifdef MINIZ_STATIC_DEFINE
+#define MINIZ_EXPORT
+#define MINIZ_NO_EXPORT
+#else
+#ifndef MINIZ_EXPORT
+#ifdef miniz_EXPORTS
+/* We are building this library */
+#define MINIZ_EXPORT
+#else
+/* We are using this library */
+#define MINIZ_EXPORT
+#endif
+#endif
+
+#ifndef MINIZ_NO_EXPORT
+#define MINIZ_NO_EXPORT
+#endif
+#endif
```

Comment:
This file gets automatically generated at build time when you use the upstream CMakeLists.txt. So this file is just a copy/paste of that generated file (since we're not using their CMakeLists.txt).

I'll add the comments anyway, since I agree with you.